src/DIRAC/FrameworkSystem/private/authorization/utils/Clients.py

```diff
@@ -0,0 +1,63 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import six
+import time
+import pprint
+
+from DIRAC import gConfig, gLogger
+from authlib.oauth2.rfc6749.util import scope_to_list, list_to_scope
+from authlib.integrations.sqla_oauth2 import OAuth2ClientMixin
+from DIRAC.ConfigurationSystem.Client.Utilities import getAuthorizationServerMetadata
+
+__RCSID__ = "$Id$"
+
+DEFAULT_CLIENTS = {
+    'DIRACCLI': dict(client_id='DIRAC_CLI', scope='proxy g: lifetime:', response_types=['device'],
+                     grant_types=['urn:ietf:params:oauth:grant-type:device_code', 'refresh_token'],
```

Comment:
There is a grep for the documentation output looking for `:param` because that so far always pointed to a syntax issue in the function parameter description. with this however we also find a legitimate occurance of `:param`

Reply:
IIUC that means it needs to be fixed in the docs test

Reply:
yes